client/src/js/leafletComponents/resultItem.js

```diff
@@ -13,9 +13,11 @@ var localization = require('../localization');
 var resultItem = {
   create: function (options) {
     this._options = options;
+    var colCount = options.isSearchResult ? 'col-xs-9 result'  : 'col-xs-6 c-result-item';
```

Comment:
There isn't anyway you would know this, but staying with the convention the css class names for this would be something like
`.c-result-item--selected` & `.c-result-item--search`